src/Network/Network/Cortex/VirtualHub/UpdateAzureRmVirtualHubCommand.cs

```diff
@@ -103,6 +103,8 @@ public class UpdateAzureRmVirtualHubCommand : VirtualHubBaseCmdlet
         [PSArgumentCompleter("Basic", "Standard")]
         public string Sku { get; set; }
 
+        public const String PreferredGWChangeDesc = "PreferredRoutingGateway parameter is deprecated. Use *HubRoutingPreference* property";
```

Comment:
```suggestion
        public const String PreferredGWChangeDesc = "PreferredRoutingGateway parameter will be deprecated. Use *HubRoutingPreference* parameter";
```
We want to be precise in breaking change messages.